src/components/Filters/index.ts

```diff
@@ -1 +1,2 @@
+export * as Components from './components';
```

Comment:
Hmmmm, wouldn't this be imported as `import { Components } from `@securityscorecard/design-system`? I'd suggest naming it with some Filter prefix

Reply:
I expect it should be exported like 
```
import { Filters } from '@securityscorecard/design-system'
const { Components } = Filters
```
@ajkl2533 But `import { Components } from '@securityscorecard/design-system/Filters'` should also work? Not to import whole Filters. I could move it one level up to make it easier to use

Reply:
Nope it will be exactly as @vaclav-zeman wrote :D 
```suggestion
export * as FilterInputs from './components';
```

Reply:
I have no idea how to achieve that `import { Components } from '@securityscorecard/design-system/Filters'` way if you have any suggestion let me know,

Reply:
I see! Will rename `Components`